### PR TITLE
Tech debt/api calls unit tests

### PR DESCRIPTION
**Pull request by aljcalandra**
This merge isn't in need of a release and there is no documentation change

### CODE REVIEW

File: test/API/test_calls.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
+import os
+import sys
+sys.path.insert(0, os.path.abspath( os.path.join(os.path.dirname(__file__), '../../src/') ))
```
Comment (m-hintz-42) [inline test/API/test_calls.py]: You add the file to path? 

Why is this needed?

File: test/API/test_calls.py [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+        self.assertRaises(TypeError, library.call_get_request)
+
+    @patch('RequestsLibrary.RequestsKeywords.create_session')
+    @patch('RequestsLibrary.RequestsKeywords.get_request')
+    def test_basic_get(self, create_session, get_request):
+        library = APILibrary()
+        r = library.call_get_request({"a": "Text"}, "Endpoint", "fullstring")
+        type(r).text = PropertyMock(return_value="success")
+        type(r).status_code = PropertyMock(return_value=200)
+        assert r.text
```
Comment (m-hintz-42) [inline test/API/test_calls.py]: Shouldn't this be `assert r.text == "success"`

If so - multiple occurrences throughout file.

File: test/API/test_calls.py (hunk 1 above)
Comment (aljcalandra) [reply test/API/test_calls.py]: Since we don't have __init__ files all over the place this is an alternative, we just add the src directory to the path.
I'd prefer this to putting init files everywhere but that is only my opinion.

File: test/API/test_calls.py (hunk 2 above)
Comment (aljcalandra) [reply test/API/test_calls.py]: yea, I totally forgot about this one
